### PR TITLE
`[PRD]` | Update withdraw modal UI

### LINKED ISSUE #2527: Update Claim payments
Looks like this will require significant redesign?
https://www.notion.so/hulking-cosmos-3e8/Q4-2024-QA-Requirements-13e2ee9c5bc280a88458e7a935d17488?pvs=4#13e2ee9c5bc28095b5f1db4b81532500

### PR DESCRIPTION
**Pull request by DarksightKellar**
Closes #2527 
Screenshot 2024-11-28 at 16 53 59

Screenshot 2024-11-28 at 16 54 19

### CODE REVIEW

**Review by mudrila — APPROVED** [review]

File: src/components/ui/modals/DelegateModal.tsx [hunk 1]
```diff
@@ -226,7 +226,7 @@ export function DelegateModal({ close }: { close: Function }) {
             </Field>
             <Button
               type="submit"
-              marginTop="2rem"
+              marginTop="3rem"
```
Comment (Da-Colon) [inline src/components/ui/modals/DelegateModal.tsx:229]: random :D

**Review by Da-Colon — CHANGES_REQUESTED** [review]
image

Looks like the top line is slightly off-centered

**Comment by nicolaus-sherrill** [comment]
Hey @DarksightKellar I have a few design callouts to what I am seeing in your screenshot:

- Text needs to be aligned to baseline.
- Every token/network/avatar circle should be 24px
- Button spacing needs to be reduced
- Apply card dimension effects
Screenshot 2024-12-03 at 3 10 20 PM

**Comment by nicolaus-sherrill** [comment]
@decentdao/engineering is it appropriate to truncate the wallet address in this modal or is it vital to see the entire address?

**Comment by DarksightKellar** [comment]
> Hey @DarksightKellar I have a few design callouts to what I am seeing in your screenshot:
> 
> * Text needs to be aligned to baseline.
> * Every token/network/avatar circle should be 24px
> * Button spacing needs to be reduced
> * Apply card dimension effects
> 
> Screenshot 2024-12-03 at 3 10 20 PM

@nicolaus-sherrill See if the latest update matches expectation

**Comment by nicolaus-sherrill** [comment]
@DarksightKellar looks good sir

**Comment by parkermccurley** [comment]
Looks great! Approved

**Review by Da-Colon — APPROVED** [review]

**Pull request merged**